aya/src/bpf.rs

```diff
@@ -196,22 +196,26 @@ pub struct BpfLoader<'a> {
     verifier_log_level: VerifierLogLevel,
 }
 
-/// Used to set the verifier log level in [BpfLoader](BpfLoader::verifier_log_level()).
-#[repr(u32)]
-#[non_exhaustive]
-#[derive(Debug, Clone, Copy)]
-pub enum VerifierLogLevel {
-    /// Disable all logging.
-    Disable = 0,
-
-    /// Default level of logging, shows verifier stats.
-    Default = 4,
-
-    /// Prints verbose logs showing tracing.
-    Verbose = 1,
+bitflags! {
+    /// Used to set the verifier log level flags in [BpfLoader](BpfLoader::verifier_log_level()).
+    pub struct VerifierLogLevel: u32 {
+        /// Sets no verifier logging.
+        const DISABLE = 0;
+        /// Enables debug verifier logging.
+        const DEBUG = 1;
+        /// Enables verbose verifier logging.
+        const VERBOSE = 2 | Self::DEBUG.bits;
+        /// Enables verifier stats
```

Comment:
Missing period at the end of the sentence

Reply:
@conectado oops just noticed this - could you fix it pls?

Reply:
sorry didn't see this message, right away